frontend/packages/kubevirt-plugin/src/utils/validations/vm/disk.ts

```diff
@@ -129,8 +133,10 @@ export const validateDisk = (
       addRequired(pvcName);
       validations.pvc = validatePVCName(pvcName, usedPVCNames);
     }
-  }
 
+    addRequired(allowedBusses.has(disk.getDiskBus()));
```

Comment:
the disk condition should still be here or we could get stuck in cdroms

Reply:
what are you suggesting?:
1. remove the line
2. keep the line and add a comment above for a followup
3. comment the line and add a comment about a followup